packages/components/table/src/table/utils-helper.ts

```diff
@@ -8,8 +8,12 @@ function useUtils<T>(store: Store<T>) {
   const getSelectionRows = () => {
     return store.getSelectionRows()
   }
-  const toggleRowSelection = (row: T, selected?: boolean) => {
-    store.toggleRowSelection(row, selected, false)
+  const toggleRowSelection = (
+    row: T,
+    selected?: boolean,
+    ignoreSelectable = true
```

Comment:
Providing a third parameter is more flexible. Should we explain it in detail in the documentation or provide an example to show the difference between adding and not adding the parameter?